src/scripts/interaction.js

```diff
@@ -1598,6 +1610,16 @@ function Interaction(parameters, player, previousState) {
         });
 
         instance.on('click', function (event) {
+
+          if(instance.libraryInfo.machineName === "H5P.IVHotspot" && parameters.wrongAnswerParams.isWrongChoice) {
+            self.wrongAnswerClicked += 1;
+
+            if(parameters.wrongAnswerParams.wrongChoicesAmount === self.wrongAnswerClicked) {
+              self.goToTaskStart = true;
```

Comment:
Should we not navigate to the seek time in this case?
For safety I would use `self.wrongAnswerClicked >= parameters.wrongAnswerParams.wrongChoicesAmount`

If for any reason the wrongAnswerClicked(count) is increased by more than one, you don't get any problems.
Not likley to happen but it is safer code.

Reply:
Fixed